packages/api/tests/models/test-rules-model.js

```diff
@@ -123,108 +149,153 @@ test.serial('deleting a kinesis style rule removes event mappings', async (t) =>
+});
+
+test.serial('update a kinesis type rule workflow does not affect value or event source mappings', async (t) => {
+  const { kinesisRule } = t.context;
+
+  // create rule
+  await rulesModel.create(kinesisRule);
+  const rule = await rulesModel.get({ name: kinesisRule.name });
+
+  // update rule value
+  const updates = {
+    name: rule.name,
+    workflow: 'new-workflow'
+  };
+
+  const updatedRule = await rulesModel.update(rule, updates);
+
+  t.is(updatedRule.name, rule.name);
+  t.is(updatedRule.rule.value, rule.rule.value);
+
+  // Event source mappings exist and have been updated
+  t.truthy(updatedRule.rule.arn);
+  t.is(updatedRule.rule.arn, rule.rule.arn);
+  t.truthy(updatedRule.rule.logEventArn);
+  t.is(updatedRule.rule.logEventArn, rule.rule.logEventArn);
+
+  await rulesModel.delete(rule);
 });
 
-test.serial('create a kinesis type rule, using the existing event source mappings', async (t) => {
+test.serial('create a kinesis type rule, using existing event source mappings', async (t) => {
+  const { kinesisRule } = t.context;
+
   // create two rules with same value
-  const rules = new models.Rule();
-  const newKinesisRule = Object.assign({}, kinesisRule);
-  newKinesisRule.rule = Object.assign({}, kinesisRule.rule);
-  newKinesisRule.name = `${kinesisRule.name}_new`;
+  const newKinesisRule = {
+    ...kinesisRule,
+    name: `${kinesisRule.name}_new`,
+    rule: {
+      ...kinesisRule.rule
+    }
+  };
 
-  await rules.create(kinesisRule);
-  const rule = await rules.get({ name: kinesisRule.name });
+  await rulesModel.create(kinesisRule);
+  const rule = await rulesModel.get({ name: kinesisRule.name });
 
-  await rules.create(newKinesisRule);
-  const newRule = await rules.get({ name: newKinesisRule.name });
+  await rulesModel.create(newKinesisRule);
+  const newRule = await rulesModel.get({ name: newKinesisRule.name });
 
   t.not(newRule.name, rule.name);
   t.is(newRule.rule.value, rule.rule.value);
   t.false(newRule.rule.arn === undefined);
   t.false(newRule.rule.logEventArn === undefined);
-  // same event source mapping
+  // Event source mappings have not changed
   t.is(newRule.rule.arn, rule.rule.arn);
   t.is(newRule.rule.logEventArn, rule.rule.logEventArn);
 
-  await rules.delete(rule);
-  await rules.delete(newRule);
+  await rulesModel.delete(rule);
+  await rulesModel.delete(newRule);
 });
 
 test.serial('it does not delete event source mappings if they exist for other rules', async (t) => {
+  const { kinesisRule } = t.context;
+
   // we have three rules to create
-  const kinesisRuleTwo = Object.assign({}, kinesisRule);
-  kinesisRuleTwo.rule = Object.assign({}, kinesisRule.rule);
-  kinesisRuleTwo.name = `${kinesisRule.name}_two`;
-  const kinesisRuleThree = Object.assign({}, kinesisRule);
-  kinesisRuleThree.rule = Object.assign({}, kinesisRule.rule);
-  kinesisRuleThree.name = `${kinesisRule.name}_three`;
-
-  const rules = new models.Rule();
+  const kinesisRuleTwo = {
```

Comment:
👍  good change.   Consider lodash.cloneDeep for style/readability  reasons if no other technical considerations require, but I'm fine either way.          This should not hold up approval.